deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
+if [ -z "$AVAIL_BOX" ]; then
+  vagrant box add chef/centos-7.0 --provider virtualbox
+fi
+
+echo "Generating 'tachyon-dev.box' based on box '$BASE_BOX' ..."
+vagrant box repackage $BASE_BOX virtualbox 1.0
+mv package.box tachyon-dev.box
```

Comment:
@chen-tian @uronce-cc  I am guessing it is to save the downloaded file, so it no longer needs to be downloaded when another vm is launched. Right now there is no logic to take advantage of those cached files though.  need to add that..